tests/nnstreamer_filter_armnn/unittest_filter_armnn.cpp

```diff
@@ -220,6 +224,125 @@ TEST (nnstreamer_filter_armnn, invoke_basic)
   sp->close (&prop, &data);
 }
 
+/**
+ * @brief get argmax from the array
+ */
+size_t get_argmax (guint8 *array, size_t size)
+{
+  size_t idx, max_idx = 0;
+  guint8 max_value = 0;
+  for (idx = 0; idx < size; idx ++)
+    if (max_value < array[idx])
+      max_idx = idx;
```

Comment:
I think you need to update `max_value` as well.

Reply:
Oh, I wonder how this unit-test worked without this.
Updated

Reply:
Amazing :)